### ISSUE TITLE
add accessible name to button in System Tray

### ISSUE THREAD

**Issue by robfentress** [Code52/carnac#218]
I am doing user testing with a blind user who uses the JAWS screen reader and would like to use Carnac to display the keys he is pressing to activate screen reader commands.  However, he cannot reliably exit the program independently, because the Carnac button in the System Tray doesn't have an accessible name that can be voiced by a screen reader.  Instead. JAWS simply announces "button" when it comes to that control.  I am not a Windows programmer, so I can't tell you the details, but there are ways to provide text labels for graphical buttons that will be voiced by screen readers.  Carnac could be a great tool for those who wish to do remote user testing with blind users, if you could fix this.
